packages/node/src/integrations/node-fetch.ts

```diff
@@ -80,7 +50,7 @@ const _nativeNodeFetchIntegration = ((options: NodeFetchOptions = {}) => {
 
           // If tracing is disabled, we still want to propagate traces
           // So we do that manually here, matching what the instrumentation does otherwise
-          if (tracingDisabled) {
+          if (!hasTracingEnabled()) {
```

Comment:
Just to verify, is this necessary/works here? IIRC this is not necessary in http, but this kind of depends on internals (how/when is what kind of span started/sampled, ...)

Reply:
Will check!

Reply:
Yep, you're correct, this is no longer needed!